src/vcState.h

```diff
@@ -173,7 +174,7 @@ struct vcState
   vcDocks changeActiveDock;
 
   bool getGeo;
-  udGeoZone *pGotGeo;
+  vcModel *pGotGeo;
```

Comment:
I'm not sure I understand why this was changed?

Reply:
Changing this type is wrong. Lots of other things are sources for this and I already have changes that change this to support more types.